src/main/java/net/mcreator/ui/workspace/selector/WorkspaceSelector.java

```diff
@@ -77,12 +79,10 @@
 public final class WorkspaceSelector extends JFrame implements DropTargetListener, INotificationConsumer {
 
 	private static final Logger LOG = LogManager.getLogger("Workspace Selector");
-
+	private static final Gson gson = new GsonBuilder().setPrettyPrinting().setStrictness(Strictness.LENIENT).create();

```

Comment:
Please revert this change that is completely unrelated to this PR

Reply:
I thought you did it, but later I found out it was changed by Idea